parser/ParseStatement.hpp

```diff
@@ -590,10 +631,86 @@ class ParseStatementInsert : public ParseStatement {
   }
 
  private:
-  std::unique_ptr<ParseString> relation_name_;
   std::unique_ptr<PtrList<ParseScalarLiteral> > literal_values_;
 
-  DISALLOW_COPY_AND_ASSIGN(ParseStatementInsert);
+  DISALLOW_COPY_AND_ASSIGN(ParseStatementInsertTuple);
+};
+
```

Comment:
Agreed with @jianqiao -- after this closes, lets do a separate PR for the two-line style.